packages/appium/lib/cli/parser.js

```diff
@@ -97,8 +99,17 @@ class ArgParser {
     }
 
     try {
-      const parsed = this.parser.parse_args(args);
-      return ArgParser._transformParsedArgs(parsed);
+      const parsed = this.parser.parse_known_args(args);
+      const [knownArgs, unknownArgs] = parsed;
+      if (
+        (unknownArgs?.length && knownArgs.driverCommand === 'run') ||
+        knownArgs.pluginCommand === 'run'
```

Comment:
is this boolean check correct? seems weird for us not to care about `unknownArgs.length` if we have a plugin command?

Reply:
I think the parens are in the wrong place.